auth/public/sdk/wapi.js

```diff
@@ -319,7 +319,13 @@ if (typeof wapiInit === "undefined") {
     };
 
     //register the app
-    axios.post('https://api.web10.app/register_app', { "url": window.location.href.split('?')[0] })
+    //axios.post('https://api.web10.app/register_app', { "url": window.location.href.split('?')[0] })
+    console.log("hello");
+    axios.post('http://api.localhost/register_app', { "url": window.location.href.split('?')[0] })
```

Comment:
we should make this URL http://api.localhost an input parameter into the wapiInit function.
it should actually be a list instead of just one, so the app can post request to register with multiple web10 providers.

wapiInit( ...,..,.., [registrationURLs])

Reply:
in other words, changing it so it will do post requests to multiple web10 providers

Reply:
Hi Jacob! Would I just make an arraylist with the http://api.localhost url and pass it in as authUrl? I'm not 100% sure how to approach this yet and what other web10 providers I'd need to list.